src/main/java/com/ss/training/lms/service/admin/AdminOverrideLoanService.java

```diff
@@ -21,10 +22,22 @@
 	@Autowired
 	BookLoanDAO loanDAO;
 
-    public void addAWeekToALoan(BookLoan loan) throws SQLException{
+    public boolean addAWeekToALoan(BookLoan loan) throws SQLException{
         Connection conn = null;
         try {
             conn = connUtil.getConnection();
+            List<BookLoan> loans = loanDAO.readAllLoansFromABorrower(loan.getCardNo(), conn);
```

Comment:
Not super relevant, but when we get a UI hopefully we can pass the loan data including the DateOut so that we don't have to do this search and check. I get that we need it with the API version though.